<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>946</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>华文中宋</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>581</width>
      <height>801</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="LINK">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>70</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文中宋</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>链地址法</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文中宋</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>哈希表-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LINEAR">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>70</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>线性探测</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DOUBLE">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>170</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>二次探测</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RANDOM">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>170</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>伪随机探测</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>350</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="SEARCH">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>420</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>查找</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>270</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>↑请选择冲突处理方法 ↑</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>520</y>
      <width>301</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="INPUT">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>420</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>输入姓名</string>
    </property>
   </widget>
   <widget class="QLabel" name="ASL_label">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文中宋</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>平均查找长度ASL=</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
